File: ruby/getting-localized-content/GetLocalizedItems.rb
# DocSection: getting_localized_content_language
# Tip: Find more about Ruby SDKs at https://docs.kontent.ai/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '8d20758c-d74c-4f59-ae04-ee928c0816b'
delivery_client.item('about_us')
               .language('es-ES')
               .execute do |response|
                 item = response.item
               end
# EndDocSection

File: ts/getting-localized-content/GetLocalizedItems.ts
// DocSection: getting_localized_content_language
// Tip: Find more about JS/TS SDKs at https://docs.kontent.ai/javascript
import { DeliveryClient, TypeResolver } from '@kentico/kontent-delivery';
import { Article } from './models/Article';

const deliveryClient = new DeliveryClient({
  projectId: '8d20758c-d74c-4f59-ae04-ee928c0816b7',
  typeResolvers: [
    // Create strongly typed models according to https://docs.kontent.ai/strongly-typed-models
    new TypeResolver('article', (rawData) => new Article)
  ]
});

deliveryClient.item<Article>('about_us')
  .languageParameter('es-ES')
  .toObservable()
  .subscribe(response => console.log(response.item));
// EndDocSection